packages/vscode-extension/package.json

```diff
@@ -830,6 +830,60 @@
         "scopeName": "teamsfx-toolkit.output",
         "path": "./syntaxes/teamsfx-toolkit-output.tmLanguage"
       }
+    ],
+    "walkthroughs": [
+      {
+        "id": "teamsToolkitQuickStart",
+        "title": "Get started with Teams toolkit  ",
+        "description": "Jumpstart your Teams app development experience   ",
+        "steps": [
+          {
+            "id": "teamsToolkitCapailities",
+            "title": "What are Teams app \"Capabilities\"",
+            "description": "[Capabilities](https://docs.microsoft.com/en-us/microsoftteams/platform/concepts/capabilities-overview) are the extension points for building apps on the Microsoft Teams platform.\n[Watch video (<1 min)](https://aka.ms/teamsfx-video)",
+            "media": {
+              "svg": "media/learn.svg",
+              "altText": "Watch"
+            }
+          },
+          {
+            "id": "teamsToolkitCommands",
+            "title": "Explore Teams toolkit commands",
+            "description": "[Open Command Palette (Ctrl+Shift+P)](command:workbench.action.quickOpen?%5B%22%3ETeams%22%5D) and type 'Teams' to find Teams toolkit commands. You can also discover [Teams toolkit icon](https://www.bing.com) on the activity bar to easily see the commands after a project creation. \n[Display Teams commands](command:workbench.action.quickOpen?%5B%22%3ETeams%22%5D)",
```

Comment:
@jaeyonglee8687 do you think we need this String here? because if users can see quick start, in most case, they already have sidebar welcome open. I think we can put "open teams toolkit icon" after users creating a project.